.travis.yml

```diff
@@ -19,6 +19,7 @@ env:
   - TEST_TYPE=admin-e2e-3
   - TEST_TYPE=pupil-e2e-1
   - TEST_TYPE=pupil-e2e-2
+  - OVERRIDE_PIN_EXPIRY="true"
 global:
   - STD_LOG_FILE="./build.log"
   - CXX=g++-4.8
```

Comment:
this file is clearly out of date because these have been removed.  the feb trial is already up to date with master so are you working from a stale codebase?

Reply:
@GuyHarwood this is a branch of the feb-trial branch which I always update before creating a new branch from it, not sure how this got in here.

Reply:
ive just checked the feb trial travis file and its up to date.

i would do a pull on the feb trial branch and merge it back into your PR branch, and also double check that you definitely created the PR branch from the feb trial branch

Reply:
if that all goes to plan you shouldnt need to modify the travis file at all.  give me a shout if you need another pair of eyes on it